xbmc/interfaces/legacy/Control.h

```diff
@@ -744,8 +744,9 @@ namespace XBMCAddon
     /// | XBFONT_RIGHT      | 0x00000001 | Align X right
     /// | XBFONT_CENTER_X   | 0x00000002 | Align X center
     /// | XBFONT_CENTER_Y   | 0x00000004 | Align Y center
-    /// | XBFONT_TRUNCATED  | 0x00000008 | Truncated text
-    /// | XBFONT_JUSTIFIED  | 0x00000010 | Justify text
+    /// | XBFONT_TRUNCATED  | 0x00000008 | Truncated text from right
+    /// | XBFONT_JUSTIFIED  | 0x00000016 | Justify text
+    /// | XBFONT_TRUNCATED_LEFT | 0x00000032 | Truncated text from left
```

Comment:
0xXXXXX means a hex constant, but the values are decimal here.
0x10 was correct for justified and truncated left should be 0x20
Same for the other similar changes of the file.

Reply:
done, thanks i didnt realized